test/cppsuite/test_harness/workload/workload_validation.h

```diff
@@ -47,22 +51,24 @@ class workload_validation {
     /*
      * Validate the on disk data against what has been tracked during the test. This is done by
      * replaying the tracked operations so a representation in memory of the collections is created.
-     * This representation is then compared to what is on disk. operation_table_name: collection
-     * that contains all the operations about the key/value pairs in the different collections used
-     * during the test. schema_table_name: collection that contains all the operations about the
-     * creation or deletion of collections during the test.
+     * This representation is then compared to what is on disk.
+     *
+     *   -  operation_table_name: Table that contains all the operations performed on keys.
```

Comment:
Trailing space before the dot point. And also two spaces between the dot point and the name.
Our dot points in comments usually just look like:

```
* - blah
```

Reply:
8153016